compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEPropagateDataLayout.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree-amd-aie/Transforms/PassDetail.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "mlir/Dialect/Linalg/Transforms/Transforms.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+#define DEBUG_TYPE "iree-amdaie-propagate-pad"
```

Comment:
Can we have a common pass name?

Currently, I see that :-
DEBUG_TYPE = `iree-amdaie-propagate-pad`
.td = `iree-amdaie-propagate-data-layout`
Test file name = `datalayout_propagation`

It'd be nice to have a common name - helps debugging/navigating eventually.